src/lib/features/events/event-store.ts

```diff
@@ -196,7 +198,14 @@ class EventStore implements IEventStore {
             .max('id')
             .where((builder) =>
                 builder
-                    .whereNotNull('feature_name')
+                    .andWhere((inner) =>
+                        inner
+                            .whereNotNull('feature_name')
+                            .whereNotIn('type', [
+                                FEATURE_CREATED,
+                                FEATURE_TAGGED,
+                            ]),
+                    )
```

Comment:
Doesn't this work the same way? 
```
        const row = await this.db(TABLE)
            .max('id')
            .where((builder) =>
                builder
                    .whereNotNull('feature_name')
                    .orWhereIn('type', [
                        SEGMENT_UPDATED,
                        FEATURE_IMPORT,
                        FEATURES_IMPORTED,
                    ]),
            )
            .andHavingNotIn('type', [FEATURE_CREATED, FEATURE_TAGGED])
            .andWhere('id', '>=', largerThan)
            .first();
```

Reply:
Also, note we use a similar query for delta calculation (method `getRevisionRange` below), maybe both would have to share some code?

Reply:
Yes, in theory that would do the same, just wanted to keep all filtering in same location. Hmm, but the code is a more readable.

Reply:
Updated getRevisionRange also.

Reply:
Actually I reverted this, as where not does not work with lists and this broke the code.